src/style.css

```diff
@@ -0,0 +1,20 @@
+body {
+  font-family: Arial, sans-serif;
+  font-size: 13px;
+}
+
+h1 {
```

Comment:
Try to avoid stylizing tags. Always use classes for this. Tags stylizing is OK when you reset default browser styles, like margin: 0 for BODY tag, which has 8px margins. Setting width and height for all DIV tags is bad practice

Reply:
Я исправил body на html.
Если использовать классы для h1, h2, p, то BackStop JS выдает ошибку. Я знаю, как работать с классами, однако нормально добавить их невозможно, поскольку автоматическая проверка не будет пройдена :(

Reply:
О, BackStop JS нормально заработал. Все изменил на классы. Теперь все, как надо :)